src/test/scala/edu/ie3/simona/model/participant/load/FixedLoadModelSpec.scala

```diff
@@ -0,0 +1,150 @@
+/*
+ * © 2024. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.participant.load
+
+import edu.ie3.simona.model.SystemComponent
+import edu.ie3.simona.model.participant.ModelState
+import edu.ie3.simona.model.participant.control.QControl
+import edu.ie3.simona.test.common.input.LoadInputTestData
+import edu.ie3.simona.model.participant.load.LoadReference.{
+  ActivePower,
+  EnergyConsumption,
+}
+import edu.ie3.simona.test.common.UnitSpec
+import edu.ie3.util.TimeUtil
+import edu.ie3.util.quantities.PowerSystemUnits
+import org.scalatest.prop.TableDrivenPropertyChecks
+import squants.Power
+import squants.energy.{KilowattHours, Kilowatts, Watts}
+
+class FixedLoadModelSpec
+    extends UnitSpec
+    with LoadInputTestData
+    with TableDrivenPropertyChecks {
+
+  private implicit val tolerance: Power = Watts(1d)
```

Comment:
Maybe not needed any longer.

Reply:
approximate() requires a tolerance as i understand it.

Reply:
Yes, thats true. Was still at the stage of should Be ... 

Good catch!